lib/AdminPages/Pages/NewsletterEditor.php

```diff
@@ -54,29 +54,38 @@ function render() {
     if (
       $woocommerce_template_id
       && $newsletter_id === $woocommerce_template_id
-      && (
-        !$this->woocommerce_helper->isWooCommerceActive()
-        || !(bool)$this->settings->get('woocommerce.use_mailpoet_editor', false)
-      )
+      && !$this->woocommerce_helper->isWooCommerceActive()
     ) {
-      header('Location: admin.php?page=mailpoet-settings&enable-customizer-notice#woocommerce', true, 302);
+      $location = 'admin.php?page=mailpoet-settings&enable-customizer-notice#woocommerce';
```

Comment:
From the specs
> If the user tries to access the template editor URL, redirect him to `?page=mailpoet-newsletters` and ...

But you redirect to settings page here.

Reply:
I guess it's a typo in specs, we changed them to redirect to the message which is in the Settings (see last 3 comments in the Jira task).

Reply:
I see, that makes sense.